ga4gh/datamodel/references.py

```diff
@@ -6,8 +6,75 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
+import glob
+
+import pysam
 
 import ga4gh.protocol as protocol
 
-# TODO use something to translate reference data in FASTA format into
-# GA4GH native objects.
+
+class ReferenceSet(object):
+    """
+    Class representing a reference set in the GA4GH datamodel. A reference
+    set consists of a directory containing a set of indexed FASTA files.
```

Comment:
Your comment here describes the class in terms of an implementation, while your comment for Reference describes it in terms of the application domain. This makes it harder for me to understand the relationship.
A scientist shouldn't have to care about the directory full of indexed FASTA; what do *they* think the ReferenceSet is? Or is it just semantic noise we should be hiding from them?